ci/certification/dummy.yml

```diff
@@ -0,0 +1,238 @@
+.certification:dummy:
+  stage: certification-dummy
+  extends: .certification-generic
+  variables:
+    DEVICE_UNDER_TEST: "prplmesh"
+  needs:
+    - job: build-in-docker
+
+MAP-5.3.1:dummy:
+  extends: .certification:dummy
+  rules:
+    # If the last commit description contains the test name and we're not on master, run it.
+    # Otherwise, make it a manual job
+    - if: '$CI_COMMIT_DESCRIPTION =~ /.*MAP-5.3.1:dummy.*/ && $CI_COMMIT_REF_NAME != "master"'
+      when: on_success
+    - when: manual
+      allow_failure: true
```

Comment:
This probably betrays my general gitlab ignorance, but:
Why allow failure? Don't we *want* a test failure to block the PR?

Reply:
Allow failure is for the manual jobs only. I'm not sure what the reasoning behind it is, but manual jobs that have allow_failure=false make the whole pipeline appear as "blocked" and need the jobs to be triggered before the pipeline is reported as "passing".

For the jobs that run automatically (so if the test is mentioned in the commit message) PRs will block until the test passes (even after an automatic rebase).
So I think for PRs we should encourage mentioning every potentially affected test names in commits messages, and use the manual jobs for the other cases (sporadically check that a test still passes for example, or check the status of a not-yet-passing test).

Reply:
> Allow failure is for the manual jobs only. I'm not sure what the reasoning behind it is, but manual jobs that have allow_failure=false make the whole pipeline appear as "blocked" and need the jobs to be triggered before the pipeline is reported as "passing".

That's a bit of historical accident...

Originally, the "manual" was meant to do block (part of) a pipeline and require manual intervention to launch it, as a kind of "approval" step. For example, you could have a job "review" that needs to be clicked by a human to allow the pipeline to succeed.

Using "manual" for the use case we have here is something that was only thought of afterwards, and they used the already existing "allow_failure" tag to mark it.